README.md

```diff
@@ -114,9 +120,13 @@ const report = await orchestrator.run()
 console.log(report)
 ```
 
-### How to set networking:
+### Networking and tests:
+
+By default tryorama assumes un-bootstrapped `Quic` networking.
```

Comment:
This is claimed, but I don't see how this is actually true yet, it doesn't appear that a default has been set as such

Reply:
Hah!  You are right.  I did that in the tests, not in the actual code!